src/_data/platforms.yml

```diff
@@ -1,50 +1,50 @@
 - platform: 'Android SDK'
   chipsets: 'x64, Arm32, Arm64'
   supported: '21 to 34'
-  besteffort: 'N/A'
+  ci-tested: '21 to 34'
   unsupported: '20 and earlier'
 - platform: 'iOS'
   chipsets: 'Arm64'
-  supported: '17'
-  besteffort: '12 to 16, 18'
+  supported: '12 to 18'
+  ci-tested: '16, 17'
   unsupported: '11 and earlier'
 - platform: 'macOS'
   chipsets: 'x64, Arm64'
-  supported: '<abbr title="Ventura">13</abbr>, <abbr title="Sonoma">14</abbr>'
-  besteffort: '<abbr title="Mojave">10.14</abbr> to <abbr title="Monterey">12</abbr>, <abbr title="Sequoia">15</abbr>'
-  unsupported: '<abbr title="High Sierra">10.13</abbr> and earlier'
+  supported: '10.14 to 10.15'
+  ci-tested: '12'
+  unsupported: '10.13 and earlier'
 - platform: 'Windows'
   chipsets: 'x64, Arm64'
   supported: '10'
-  besteffort: '11'
+  ci-tested: 
   unsupported: '8 and earlier'
 - platform: 'Debian (Linux)'
   chipsets: 'x64, Arm64'
   supported: '11, 12'
-  besteffort: '10'
+  ci-tested: 
   unsupported: '9 and earlier'
 - platform: 'Ubuntu (Linux)'
   chipsets: 'x64, Arm64'
   supported: '20.04 LTS, 22.04 LTS'
```

Comment:
If this now includes "best effort" then "20.04 to 24.04 LTS"

Reply:
No, I don't think so.